tests/lib/rules/no-tabs.js

```diff
@@ -24,10 +24,13 @@ ruleTester.run("no-tabs", rule, {
         "function test(){\n" +
         "  //   sdfdsf \n" +
         "}",
+        "var str = 'test\tstring'",
+        "var re = /test\tstring/"
```

Comment:
Ah, if it replaces a tab by `\t` escape sequence, it does not change semantics. I'm OK in that case.